src/frontend/src/instance/distributed.rs

```diff
@@ -121,6 +121,10 @@ impl DistInstance {
         info!("Successfully created distributed table '{table_name}' with table id {table_id}");
 
         table_info.ident.table_id = table_id;
+
+        // TODO: Fetch a latest region_numbers from MetaSrv so that FrontCatalogManager can use a latest one.
```

Comment:
Normally we don't put `regions = N` in `with` for distributed table. Actually the part regarding "regions" in the create distributed table sql might just be changed in the future (see https://github.com/GreptimeTeam/greptimedb/issues/1281). So I think we can ignore the "regions" table options here, and test other table options.

The other table options to test are: "ttl" and "write_buffer_size". See https://docs.greptime.com/reference/sql/create#table-options. Also for create external table, there are more table options: https://docs.greptime.com/reference/sql/create#create-external-table.

All in all, we'd like the output of show create table sql behaves the same in both standalone mode and distributed mode.

Reply:
>Normally we don't put regions = N in with for distributed table.

Why does `greptimedb` show `regions = ~` in `with` for standalone, like so:

https://github.com/GreptimeTeam/greptimedb/blob/9352649f22b558bf7c44f25a41b301271ea045e5/tests/cases/standalone/show/show_create.result#L33-L34

If we don't need to show `regions = ~`, I think that we can remove it not only for distributed mode but also for standalone mode.

Reply:
Yes, we can remove it right now and leave a TODO. After the new engine is ready, we can go back to add it back.

Reply:
I don't think we should add `regions = ~` in the output of show create table sql. In standalone mode, there's only one region. In distributed mode, we don't use `with (regions = ~)` to create multiple regions. @killme2008

Reply:
The standalone will support multi-regions tables in future.

Reply:
@MichaelScofield  
Should I remove this TODO ?

Reply:
@NiwakaDev yes, please. Regarding "regions = ~" in show create SQL's output, let's reconsider it after region server refactor is done. Thx!

Reply:
Addressed your review: 5939ce5